### DISCUSSION TITLE
Specifying product categories for global forecasting

### DISCUSSION THREAD

**Discussion by ngupta23** [alan-turing-institute/sktime#2713] [Q&A]
I noticed that in the [pydata berlin notebooks](https://github.com/sktime/sktime-tutorial-pydata-berlin-2022/blob/main/notebooks/3_hierarchical_global_forecasting.ipynb), the example for global forecasting uses identifiers `dept_id`, `cat_id`, `store_id`, etc. as numeric values. My question is related to the type of data - should these not be treated as categorical variables and encoded (OHE or similar) before modeling?

**Comment by fkiraly** [comment]
Hm, looks like an oversight?

You can, of course, still use a categorical/numerical sklearn pipeline inside.

FYI @danbartl (notebook author)

**Reply by ngupta23** [reply]
Hi @fkiraly If the number of categories (e.g. stores, products, etc.) is large, then the categorical variable encoding like One Hot Encoding will result in a large number of sparse features. What would be the recommended approach to encode these categorical variables in this case?
